spyder/app/tour.py

```diff
@@ -987,6 +1018,7 @@ def _process_widgets(self, names, spy_window):
 
     def _set_data(self):
         """ """

```

Comment:
I didn't write this function, but I can write it though, I just want to be sure. Should I?